meta-freescale-mbl/recipes-bsp/u-boot/u-boot-imx_%.bbappend

```diff
@@ -7,18 +7,27 @@ inherit mbl-uboot-sign
 SRCBRANCH = "imx_v2018.03_4.14.78_1.0.0_ga-mbl"
 SRC_URI = "git://git.linaro.org/landing-teams/working/mbl/u-boot.git;protocol=https;nobranch=1 \
 "
-SRC_URI_append_imx8mmevk = "file://0002-imx8mmevk-mbl-set-noenv.patch"
-
 # MBL_UBOOT_VERSION should be updated to match version pointed to by SRCREV
 MBL_UBOOT_VERSION = "2018.03"
 
 SRCREV = "e9cb2c6d8a6227a189702ab2cfc7b1273689ddb2"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/u-boot-imx:"
 
+
+do_configure_prepend_imx8mmevk-mbl() {
+    # Workaround to use merge_config.sh when using UBOOT_CONFIG variable
```

Comment:
Could you expand this comment to explain a bit more why we're adding this workaround?

Reply:
Added in 0cbcfa4